Doc/library/pdb.rst

```diff
@@ -164,6 +164,9 @@ slightly different way:
    .. versionchanged:: 3.7
       The keyword-only argument *header*.
 
+   .. versionchanged:: 3.13
+      :func:`set_trace` will enter the debugger immediately after the call,
+      rather than before the next line of code is executed.
```

Comment:
Not a huge deal, but you *might* consider rewording this (and the other two places this text is copied). A hasty reading implies that it won't break before the next line is executed ("the call" is a bit vague here).

Maybe something like "...will enter the debugger immediately, rather than on the following line."

You could even add a little color by calling out something like "This addresses a common issue where users may find the debugger starting in a different context than the original breakpoint call (such as outside of an exception handler)."

Reply:
I'm open to all documentation related suggestions as I'm not the expert of English. It would be nice to have the reason for the change in the news entry (whatsnew), but should the documentation mention it as well?

Reply:
I'd keep the `versionchanged` directive brief and unambiguous, but feel free to go nuts on What's New. Maybe even include a code example with an exception handler showing the old behavior vs the new behavior (like trying to print `e` or something).

Reply:
How about 

> :func:`set_trace` will enter the debugger immediately, rather than on the next line of code to be executed.

The problem of "following line" is that it makes people think about the "following line" in the source code - but that's not the case

```python
a = 1
if a > 0:
    breakpoint()
else:
    print("not here")
print("stopped here!")

Reply:
Sure, makes sense.

Reply:
I'd drop "to be" in "to be executed".

Reply:
Doesn't it feel like the debugger will be brought up after the line is executed? Rather than the fact that it is brought up before the line is executed?

Reply:
Eh, maybe. I think we're probably both overthinking it. This was just a random thought I had when reading it, since it feels weird to essentially say "it won't be right before the next line anymore". But I trust you to choose the wording you like best. :)

Reply:
Okay I added the docs for bdb, rephased it a bit. I also added some extra context on the whatsnew entry.